windows/make.vbs

```diff
@@ -1,2 +1,125 @@
+'Build script for KOLIBRI INSTALLER WINDOWS
+'
+'Steps
+'
+'1. Set Kolibri Version
+'2. Download python installer
+'3. Build the installation package
+'
+'python Variable
+'pythonVersion == must set a specific verion (e.g. "2.7", "3.4.3" and etc.)
+'python32BitLink & python64BitLink == URL of the python .exe 
+'pythonDir == Directory when python must save
+'
+'REF: VB Scripting References
+'https://www.codeproject.com/tips/506439/downloading-files-with-vbscript
+'http://www.csidata.com/custserv/onlinehelp/vbsdocs/vbs14.htm
+'https://stackoverflow.com/questions/20626863/can-you-get-the-pc-user-name-in-vbs
+'https://blogs.technet.microsoft.com/heyscriptingguy/2005/02/08/how-can-i-find-and-replace-text-in-a-text-file/
+
+
+'***************************************************************************************************************
+Function DownloadPythonInstaller(strLink, path)
+  'strLink == Get file name from URL.
+  'path == for directory file
+  strSaveName = Mid(strLink, InStrRev(strLink,"/") + 1, Len(strLink))
+  strSaveTo = Path & strSaveName
+  
+  On Error Resume Next
+  'Create an HTTP object
+  Set objHTTP = CreateObject("MSXML2.XMLHTTP")
+  Set objFSO = CreateObject("Scripting.FileSystemObject")
+  
+  If objFSO.FileExists(strSaveTo) Then

```

Comment:
Uhm, what about `If not objFSO...`? :)

Reply:
I think that it's okay to check if the file exist, but it's risky! If it exists, we would need to verify that it's written correctly. And how do we do that? Then we need to fetch a checksum etc.

So I think that if it exists, we need to delete it and start over.